webpages/profile.html

```diff
@@ -5,6 +5,50 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Profile</title>
+    <link rel="stylesheet" href="/styles/profile.css" />
```

Comment:
If you have the @import into the index.css you would only have to link the index.css here. 
Just make sure you tell the profile.html that the index.css is in a subdirectory.